### DISCUSSION TITLE
Communication with bot?

### DISCUSSION THREAD

**Discussion by vstath** [revenkroz/telegram-web-app-bot-example#17] [Q&A]
Hello,

maybe I'm missing something (and I tried a couple things already). 

How can my web app communicate with my bot? I want to press a button or something on my web app and get a notification on my bot (like I have with the inline_keyboard on normal TG buttons). 

Can you please provide some pointers? I am very new to TG bots and apps. 

BTW, thank you for your code :-)

**Comment by vstath** [comment]
I must add, I'm not talking about sending an XHR request to a backend out there. I'm talking about internal communication with the bot that triggered the web app.

**Reply by Njking124** [reply]
Same issue but It is working fine on telegram web
